src/main/java/games/strategy/triplea/ui/menubar/LobbyMenu.java

```diff
@@ -185,18 +185,8 @@ private void addBanMacAddressMenu(final JMenu parentMenu) {
       if (mac == null || mac.length() < 1) {
         return;
       }
-      final String error;
-      if (mac.length() != 28) {
-        error = "Must be 28 characters long";
-      } else if (!mac.startsWith(HASHED_MAC_PREFIX)) {
-        error = "Must start with: " + HASHED_MAC_PREFIX;
-      } else if (!mac.matches("[0-9a-zA-Z$./]+")) {
-        error = "Must use only these characters: 0-9a-zA-Z$./";
-      } else {
-        error = null;
-      }
-      if (error != null) {
-        JOptionPane.showMessageDialog(lobbyFrame, "The hashed Mac Address you entered is invalid (" + error + ").",
```

Comment:
There's some loss of fidelity in the new implementation because we don't inform the mod of the exact reason the hashed MAC address is invalid.  However, I don't think that's a big deal as we provide them with an example of a valid hashed MAC address in the same dialog.

(Ditto below for mutes.)